### PR TITLE
Update mysql-install.sh

### PR DESCRIPTION
**Pull request by davalanche**
## Description

Added Major Release selection

## Type of change
Please check the relevant option(s):

- [ ] Bug fix (non-breaking change that resolves an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (a fix or feature that would cause existing functionality to change unexpectedly)
- [ ] New script (a fully functional and thoroughly tested script or set of scripts.)
- [ ] Documentation update required (this change requires an update to the documentation)

## Prerequisites
The following efforts must be made for the PR to be considered. Please check when completed:
- [X] Self-review performed (I have reviewed my code, ensuring it follows established patterns and conventions)
- [X] Testing performed (I have tested my changes, ensuring everything works as expected)

### CODE REVIEW

File: install/mysql-install.sh [hunk 1]
```diff
@@ -24,9 +24,32 @@ $STD apt-get install -y \
   mc
 msg_ok "Installed Dependencies"
 
+PS3="Please select a Major Release: "
```
Comment (MickLesk) [inline install/mysql-install.sh:27]: Why not echo/whiptail for choosing?

File: install/mysql-install.sh [hunk 2]
```diff
@@ -24,9 +24,32 @@ $STD apt-get install -y \
...
+      RELEASE_REPO="mysql-8.0"
+      RELEASE_AUTH="mysql_native_password"
+      break
+      ;;
+    "8.4 - LTS")
+      RELEASE_REPO="mysql-8.4-lts"
+      RELEASE_AUTH="caching_sha2_password"
+      break
+      ;;
+    "9.1 - Innovation")
```
Comment (MickLesk) [inline install/mysql-install.sh:41]: Is Innovation a stable Release? The name looks like a prototype / beta

File: install/mysql-install.sh (hunk 1 above)
Comment (davalanche) [reply install/mysql-install.sh:27]: Good question.. I will look into it and submit an updated PR.

File: install/mysql-install.sh (hunk 2 above)
Comment (davalanche) [reply install/mysql-install.sh:41]: _Both Innovation and LTS are production-grade quality. If you desire access to the latest features and improvements and enjoy staying on top of the latest technologies, the MySQL Innovation releases might be the best fit for you._ [Source](https://dev.mysql.com/blog-archive/introducing-mysql-innovation-and-long-term-support-lts-versions/)

File: install/mysql-install.sh [hunk 3]
```diff
@@ -24,9 +24,32 @@ $STD apt-get install -y \
   mc
 msg_ok "Installed Dependencies"
 
+PS3="Please select a Major Release: "
+RELEASES=("8.0" "8.4 - LTS" "9.1 - Innovation")
+select RELEASE in "${RELEASES[@]}"; do
+  case ${RELEASE} in
+    "8.0")
```
Comment (havardthom) [inline install/mysql-install.sh:31]: Any particular reason to include this in addition to `8.4 - LTS`

Comment (MickLesk) [reply install/mysql-install.sh:31]: some scripts work only on MySQL 8.0.xx 🥴😅

**Pull request closed without merging**